datasets/standvastgoed/dataset.json

```diff
@@ -428,7 +412,7 @@
             "type": "string",
             "provenance": "$.ligging.omschrijving",
```

Comment:
Dit type provenance werkt alleen als de data binnenkomt als ndjson blobje. Volgens mij worden voor deze dataset toch direct tabellen gevuld in de ref.db.? In dat geval doet deze provenance niks en moet dan weggelaten worden.